cmd/cosign/cli/generate/generate_key_pair.go

```diff
@@ -85,7 +86,12 @@ func GenerateKeyPairCmd(ctx context.Context, kmsVal string, args []string) error
 		return err
 	}
 
-	if cosign.FileExists("cosign.key") {
+	fileExists, err := icos.FileExists("cosign.key")
+	if err != nil {
+		return fmt.Errorf("error checking if cosign.key exists: %w", err)
```

Comment:
small request, change the work `error` to `failed`

Reply:
Done